### PR TITLE
Changed more print statements to log statements.

### PR DESCRIPTION
**Pull request by leonardr**
The same as before. In this case I sometimes need to use the generic 'logging' library instead of self.log because the CirculationParser class, unlike Monitor, doesn't define a log object.

### CODE REVIEW

File: threem.py [hunk 1]
```diff
@@ -70,7 +71,10 @@ def get_events_between(self, start, end, cache_result=False):
         try:
             events = EventParser().process_all(response.content)
         except Exception, e:
-            print response.content
+            self.log.error(
```
Comment (courte) [inline threem.py]: Is `self.log` automatically set when you import logging?

Comment (leonardr) [inline threem.py]: No, self.log is a property of Monitor (core/monitor.py) which looks up a logger based on the monitor's .service_name. If you're not inside a Monitor (or something similar like CoverageProvider) there is no self.log.

Comment (courte) [inline threem.py]: Okay, so this self.log needs to be changed to `logging.error`. All the others are good to go.

Comment (courte) [inline threem.py]: Wait, jk, ThreeMAPI's has the log.

**Comment by courte** [comment]
Closing manually because I accidentally rebased against master locally. (Sorry! :skull:) This has been merged into master under the sha: 39cd8486ef49cb92b193d176f95acb9378a17d00.

**Pull request closed without merging**